README.md

```diff
@@ -22,5 +22,5 @@ The HUGR specification (still in draft) is [here](specification/hugr.md).
 This project is licensed under Apache License, Version 2.0 ([LICENSE][] or http://www.apache.org/licenses/LICENSE-2.0).
 
   [build_status]: https://github.com/CQCL/portgraph/workflows/Continuous%20integration/badge.svg?branch=main
-  [msrv]: https://img.shields.io/badge/rust-1.69.0%2B-blue.svg?maxAge=3600
+  [msrv]: https://img.shields.io/badge/rust-1.70.0%2B-blue.svg?maxAge=3600
```

Comment:
I have no idea what the function of the `maxAge` argument is - it could probably be removed?

Reply:
It's the max cache lifetime (see https://shields.io/). It seems they ignore anything below the default, so dropping it shouldn't change anything. Go ahead.